internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -259,28 +259,30 @@ func ParseEnv(e Env) (sdktypes.Env, error) {
 }
 
 type Trigger struct {
-	TriggerID sdktypes.UUID `gorm:"primaryKey;type:uuid;not null"`
-
+	TriggerID    sdktypes.UUID  `gorm:"primaryKey;type:uuid;not null"`
 	ProjectID    sdktypes.UUID  `gorm:"index;type:uuid;not null"`
-	SourceType   string         `gorm:"index"`
 	ConnectionID *sdktypes.UUID `gorm:"index;type:uuid"`
 	EnvID        sdktypes.UUID  `gorm:"index;type:uuid;not null"`
-	Name         string
+
+	SourceType   string `gorm:"index"`
 	EventType    string
 	Filter       string
 	CodeLocation string
 
-	// enforce foreign keys
-	Project    *Project
-	Env        *Env
-	Connection *Connection
-	Ownership  *Ownership `gorm:"polymorphic:Entity;"`
-
+	Name string
 	// Makes sure name is unique - this is the env_id with name.
-	UniqueName string `gorm:"uniqueIndex;not null"`
+	UniqueName string `gorm:"uniqueIndex;not null"` // env_id + name
```

Comment:
I added only deleted_at, all other were just slightly rearranged.

Reply:
@efiShtain that's a thing i did (foolishly?) long ago. I'm ok with killing it if you want to.

Reply:
Adding my 2 cents: I didn't have an opinion about trigger names when @itayd added them, because they weren't seen anywhere. But now they **are** visible (and important) when creating webhook triggers, so I prefer to keep them as-is. Auto-generating a meaningful name based on a trigger's details would be complicated and sensitive (for example, you can have 2 triggers with the same settings except a different complicated `filter` field).

Reply:
if it was here before, irrelevant to this PR
but I prefer to clean it up in a different ticket

https://linear.app/autokitteh/issue/ENG-1534/trigger-uniquename-cleanup-in-db

resolving this for now